app/models/puzzle.server.ts

```diff
@@ -3,10 +3,12 @@ import type { Coordinates, Puzzle as PrismaPuzzle } from '@prisma/client'
 import shortUUID from 'short-uuid'
 
 import { db } from '~/db.server'
-import { HTMLSanitizer } from '~/utils'
+import { HTMLSanitizer } from '~/utils/'
+
+export type CoordinatePoints = Pick<Coordinates, 'lat' | 'lng'>
```

Comment:
On peut juste mettre `Coordinates`, l'appli n'a pas à savoir qu'on a une abstraction en bdd
```suggestion
export type Coordinates = Pick<PrismaCoordinates, 'lat' | 'lng'>
```